TaylorFramework/Modules/finder/Constants.swift

```diff
@@ -6,23 +6,21 @@
 //  Copyright © 2015 YOPESO. All rights reserved.
 //
 
-import Foundation
-
 enum FinderError: ErrorType {
     case NotSpecifiedRootPath
     case NotSpecifiedType
     case WrongFilePath(path: String)
 }
 
-struct DirectorySuffix {
-    static let AllFiles = "/*"
-    static let AnyCombination = ".*"
-    static let Slash = "/"
+enum DirectorySuffix: String {
```

Comment:
Could you please either rename this enum, or make an alias for it, as it is being used sometimes to check if string starts with it which is not the purpose of a "Suffix".